Core/src/Material/Interactions.cpp

```diff
@@ -462,7 +464,7 @@ float Acts::computeEnergyLossMode(const MaterialSlab& slab, PdgParticle absPdg,
                                   float m, float qOverP, float absQ) {
   // see ATL-SOFT-PUB-2008-003 section 3 for the relative fractions
   // TODO this is inconsistent with the text of the note
-  return 0.9f * computeEnergyLossLandau(slab, m, qOverP, absQ) +
+  return 0.9f * computeEnergyLossBethe(slab, m, qOverP, absQ) +
```

Comment:
I don't have all these equations in mind currently, but I think it is worth a comment why this function is chosen to approximate this. Same for below...

Reply:
It actually comes from the references note `ATL-SOFT-PUB-2008-003` https://cds.cern.ch/record/1114577/files/soft-pub-2008-003.pdf

Reply:
Okay, so it was just wrong before?